files/SZWESAMP/ZWEIRAC

```diff
@@ -0,0 +1,292 @@
+  RDEFINE FACILITY BPX.DAEMON UACC(NONE)
+  PERMIT BPX.DAEMON CLASS(FACILITY) ACCESS(UPDATE) 0
+    ID({zowe.setup.security.users.zowe})
+
+  RLIST   FACILITY BPX.SERVER ALL
+  RDEFINE FACILITY BPX.SERVER UACC(NONE)
+  PERMIT BPX.SERVER CLASS(FACILITY) ACCESS(UPDATE) -
+    ID({zowe.setup.security.users.zowe})
+
+/* permit Zowe main server to create a user's security environment   */
+/* comment out the following 2 lines if the OMVSAPPL is not defined  */
+/* in your environment                                               */
+  PERMIT OMVSAPPL CLASS(APPL) ID({zowe.setup.security.users.zowe}) -
+    ACCESS(READ)
+  SETROPTS RACLIST(APPL) REFRESH
+
+/* permit Zowe main server to set job name                           */
+  RLIST   FACILITY BPX.JOBNAME ALL
+  RDEFINE FACILITY BPX.JOBNAME UACC(NONE)
+  PERMIT BPX.JOBNAME CLASS(FACILITY) ACCESS(READ) -
+    ID({zowe.setup.security.users.zowe})
+
+  SETROPTS RACLIST(FACILITY) REFRESH
+
+/* permit Zowe main server to use client certificate mapping service */
+  RLIST   FACILITY IRR.RUSERMAP ALL
+  RDEFINE FACILITY IRR.RUSERMAP UACC(NONE)
+  PERMIT IRR.RUSERMAP CLASS(FACILITY) ACCESS(READ) -
+    ID({zowe.setup.security.users.zowe})
+
+/* permit Zowe main server to use distributed identity mapping       */
+/* service
+  RLIST   FACILITY IRR.IDIDMAP.QUERY ALL
+  RDEFINE FACILITY IRR.IDIDMAP.QUERY UACC(NONE)
+  PERMIT IRR.IDIDMAP.QUERY CLASS(FACILITY) ACCESS(READ) -
+   ID({zowe.setup.security.users.zowe})
+
+/* permit Zowe main server to cut SMF records                        */
+  RLIST   FACILITY IRR.RAUDITX ALL
+  RDEFINE FACILITY IRR.RAUDITX UACC(NONE)
+  PERMIT IRR.RAUDITX CLASS(FACILITY) ACCESS(READ) -
+    ID({zowe.setup.security.users.zowe})
+
+  SETROPTS RACLIST(FACILITY) REFRESH
+/* show results .................................................... */
+  RLIST   FACILITY ZWES.IS           ALL
+  RLIST   FACILITY BPX.DAEMON        ALL
+  RLIST   FACILITY BPX.SERVER        ALL
+  RLIST   FACILITY BPX.JOBNAME       ALL
+  RLIST   FACILITY IRR.RUSERMAP      ALL
+  RLIST   FACILITY IRR.RAUDITX       ALL
+
+/* DEFINE ZOWE DATA SET PROTECTION ................................. */
+
+/* - HLQ..SZWEAUTH is an APF authorized data set. It is strongly     */
+/*   advised to protect it against updates.                          */
+/* - The sample commands assume that EGN (Enhanced Generic Naming)   */
+/*   is active, which allows the usage of ** to represent any number */
+/*   of qualifiers in the DATASET class. Substitute *.** with * if   */
+/*   EGN is not active on your system.                               */
+
+/* HLQ stub                                                          */
+  LISTGRP  {zowe.setup.dataset.prefix}
+  ADDGROUP {zowe.setup.dataset.prefix} DATA('Zowe - HLQ STUB')
+
+/* general data set protection                                       */
+  LISTDSD PREFIX({zowe.setup.dataset.prefix}) ALL
+  ADDSD  '{zowe.setup.dataset.prefix}.*.**' UACC(READ) DATA('Zowe')
```

Comment:
This line goes beyond col 72, if max length for `zowe.setup.dataset.prefix` is used:
```
  ADDSD  '{zowe.setup.dataset.prefix}.*.**' UACC(READ) DATA('Zowe')

----+----1----+----2----+----3----+----4----+----5----+----6----+----7----+
  ADDSD  '----+----1----+----2----+----3----+.*.**' UACC(READ) DATA('Zowe')
```

This is simple TSS test, what is col 73+ is ignored:
```
//TSS      DD DATA,DLM=$$,SYMBOLS=JCLONLY                                
----+----1----+----2----+----3----+----4----+----5----+----6----+----7---
TSS LIST(MARTIN1)                                           SEGMENT(OMVS)
$$                                                                       

SYSTSPRT:
 ...+....10...+....20...+....30...+....40...+....50...+....60...+....70.|
READY                                                                   
TSS LIST(MARTIN1)                                           SEGMENT(OMVS
TSS0249E STATEMENT CONTAINS UNBALANCED PARENTHESIS                      
READY                                                                   
END                                                                     
```

Reply:
should we fix with a SET statement then?

Reply:
I would just split the command, the dataset has max length given, this will always fit:
```
ADDSD  '{zowe.setup.dataset.prefix}.*.**' UACC(READ) - 
  DATA('Zowe')
```